pom.xml

```diff
@@ -21,7 +21,7 @@
         <groupId>org.jboss.aerogear</groupId>
         <artifactId>aerogear-parent</artifactId>
         <!-- NOTE: if updated for version of UPS dependencies, please update the 'aerogear.bom.version' as well -->
-        <version>0.2.17</version>
+        <version>0.2.17-SNAPSHOT</version>
```

Comment:
sorry - this is my fault. it should be 0.2.18-snapshot - as 0.2.17 is alreay out on maven-central

I just forgot to push the update to master (on parent) last week :/